build.sbt

```diff
@@ -55,7 +55,8 @@ lazy val commonSettings = Seq(
 
 lazy val commonDependencies = Seq(
   "com.typesafe" % "config" % "1.3.2",
-  "org.scalatest" %% "scalatest" % "3.2.0-M1" % "it, test"
+  "org.scalatest" %% "scalatest" % "3.2.0-M1" % "it, test",
+  "com.typesafe.scala-logging" %% "scala-logging" % "3.9.2"
```

Comment:
we were incorrectly relying on a transitive dependency pulling this in.  I decided to put in common, after some thought.  But anything we put in common is really hard to escape so be careful!